packages/web-ui/src/lab/Button/useButtonColorScheme.ts

```diff
@@ -0,0 +1,85 @@
+import { colors, colorsCommon } from '@utilitywarehouse/colour-system';
+
+export const useSolidButtonColorScheme = () => {
+  const commonFocusStyles = {
+    cyan: { outlineColor: colors.cyan700 },
+    red: { outlineColor: colors.red700 },
+    green: { outlineColor: colors.green700 },
+  };
+  return {
+    cyan: {
+      default: { color: colors.cyan1000, backgroundColor: colors.cyan400 },
+      hover: { backgroundColor: colors.cyan500 },
+      focus: commonFocusStyles.cyan,
+    },
+    red: {
+      default: { color: colorsCommon.brandWhite, backgroundColor: colors.red500 },
+      hover: { backgroundColor: colors.red600 },
+      focus: commonFocusStyles.red,
+    },
+    green: {
+      default: { color: colorsCommon.brandWhite, backgroundColor: colors.green500 },
+      hover: { backgroundColor: colors.green600 },
+      focus: commonFocusStyles.green,
+    },
+  };
+  // const outlineColorSchemes = {
+  //   cyan: {
+  //     solid: {
+  //       default: { color: colors.cyan1000, backgroundColor: colors.cyan400 },
+  //       hover: { backgroundColor: colors.cyan500 },
+  //     },
+  //     outline: {
+  //       default: { color: colors.cyan1000, borderColor: colors.cyan400 },
+  //       hover: { backgroundColor: colors.cyan75 },
+  //     },
+  //     common: {
+  //       focus: { outlineColor: colors.cyan700 },
```

Comment:
can this commented code be removed?

Reply:
🙄 this whole file can be removed. Thanks @thkobierecki